src/Utils/client.js

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+const API_BASE = "https://pokeapi.co/api/v2";
+const POKEMON_TOTAL = 1292; // setting limit for "all pokemon" query
+
+/////////////
+// HELPERS //
+/////////////
+
+/////////////
+// PRUNING // (since we don't want thaaaat much data)
+/////////////
+
+const pruneTypes = (typesData) => {
+  return typesData.map((t) => t.type.name);
+};
+
+const pruneSprites = (spritesData) => {
+  const prunedData = {};
+
+  // this field seemed suspect so this is a cautious measure
+  if (
+    spritesData["other"] !== undefined &&
+    spritesData["other"]["official-artwork"] !== undefined
```

Comment:
can just do if (!spritesData["other] && !......) instead of !== undefined

Reply:
`spritesData["other"]` works yup